ros_buildfarm/workspace.py

```diff
@@ -86,7 +86,9 @@ def call_build_tool(
         if colcon_verb == 'test':
             cmd += [
                 '--event-handlers', 'console_direct+',
-                '--executor', 'sequential']
+                '--executor', 'sequential',
+                # Force junit_family to make Jenkins xunit plugin 2.x happy
+                '--pytest-args', '-o', 'junit_family=xunit2']
```

Comment:
I don't think we should set this unconditionally. This should mimick the logic from ros2/ci#503 imo.

Reply:
Good point, see 9a69528